oeps/processes/oep-0055-proc-project-maintainers.rst

```diff
@@ -0,0 +1,422 @@
+===========================
+OEP-55: Project Maintainers
+===========================
+.. list-table::
+   :widths: 25 75
+
+   * - OEP
+     - :doc:`OEP-0055 <oep-0055-proc-project-maintainers>`
+   * - Title
+     - Project Maintainers
+   * - Last Modified
+     - 2022-01-28
+   * - Authors
+     - Edward Zarecor <e0d@tcril.org>, Sarina Canelake <sarina@tcril.org>
+   * - Arbiter
+     - Jeremy Bowman <jbowman@2u.com>
+   * - Status
+     - Draft
+   * - Type
+     - Process
+   * - Created
+     - 2022-01-28
+   * - Review Period
+     - <start - target end dates for review>
+   * - Resolution
+     - <links to any discussions where the final status was decided>
+   * - References
+     - <links to any other relevant discussions or relevant related materials>
+
+Abstract
+========
+
+The Open edX platform is a service oriented platform for authoring and
+delivering online learning based on the `Django framework`_.  At the
+highest level, the platform is composed of numerous services (IDAs
+[#]_), micro-front ends (MFEs [#]_) and libraries.  The code for these
+services and libraries is managed in git and hosted on GitHub
+platform.
```

Comment:
Nit: "...and hosted on the GitHub platform" or "...and hosted on GitHub's platform"

Reply:
Fixed.